test/runnable/testxmm.d

```diff
@@ -2388,6 +2388,29 @@ void test23009()
     assert(R.array == correct);
 }
 
+/*****************************************/
```

Comment:
Add a test case that covers `bytes` too. Also missing checks to see if codegen is executing reliably.

Reply:
> bytes

Not really necessary. The code says `sz <= 2`

> missing checks to see if codegen is executing reliably.

The code example in the bug report uses unsafe casts that result in memory corruption

Reply:
I mean, with the SIMD vectorized code, it can be reliably tested.

Reply:
This just got ignored 🙃🙃

Reply:
There are bytes tests present, granted there aren't any char tests, but really they are just the same thing.

Reply:
> There are bytes tests present, granted there aren't any char tests, but really they are just the same thing.

I'm talking about the codegen. Codegen is not verified. It's only testing if it is compiling, not if the actual codegen works as expected.